Dockerfile

```diff
@@ -12,10 +12,17 @@ RUN apt-get update && \
  && sed 's/# PassThroughPattern:.*this would allow.*/PassThroughPattern: .* #/' -i /etc/apt-cacher-ng/acng.conf \
  && rm -rf /var/lib/apt/lists/*
 
+ENV TINI_VERSION v0.14.0
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini /tini
```

Comment:
I would prefer the `tini` being installed in a single Dockerfile instruction, eg. https://github.com/bitnami/minideb-extras/blob/master/jessie/Dockerfile#L25-L32